src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -174,7 +171,7 @@ private void SafeOnBufferChange(object sender, TextContentChangedEventArgs e)
                 // The spans we have stored for issues relate to the previous text buffer and
                 // are no longer valid, so we need to translate them to the equivalent spans
                 // in the new text buffer.
-                UpdateDirtySpans(e);
+                currentSnapshot = e.After;
```

Comment:
This is the only line of code we need to keep from the `UpdateDirtySpans` method.